observations/obs_converters/GSI2DART/gsi_to_dart.f90

```diff
@@ -44,7 +44,7 @@ program gsi_to_dart
 ! Initialize MPI (from mpisetup module)
 ! provides nproc           process number
 ! provides numproc         total number of processes
-! provides my_communicator (mpi_comm_world)
+! provides mpi_comm_world
```

Comment:
i think the original comment here (my_communicator) is right.  it may be initialized to mpi_comm_world, but assuming that means if they create a different communicator this code will fail.  is there a problem leaving it as my_communicator?

Reply:
There is a funny thing going on here ... _I_ added the `my_communicator` as part of my testing.
At one point I tried to make mpiutils private and wanted to use the 'use, only' strategy so I could see what was going where. So ... in the end ... I backed out my changes to leave the code as unmodified as possible. Seems almost everything you like is stuff I put in, not the 'original'. Just sayin'